src/dmd/mtype.d

```diff
@@ -1131,16 +1145,9 @@ extern (C++) abstract class Type : ASTNode
         t.arrayof = null;
         t.pto = null;
         t.rto = null;
```

Comment:
What about pto and rto? Nobody outside of mtype should be touching them either.

Reply:
This is enough for the moment. Don't want to make it too big.